cloud/edgecontroller/pkg/cloudhub/channelq/channelq.go

```diff
@@ -72,9 +72,10 @@ func (q *ChannelEventQueue) dispatchMessage() {
 		}
 		resource := msg.Router.Resource
 		tokens := strings.Split(resource, "/")
+		tokenLen := len(tokens)
```

Comment:
nit: tokenLen means the length of a given token which I think it's not suitable for our case.

I suggest to use `numOfTokens` instead. Thoughts?

Reply:
nice